crates/valence_entity/build.rs

```diff
@@ -506,6 +524,22 @@ fn build() -> anyhow::Result<TokenStream> {
         }]);
     }
 
+    systems.extend([quote! {
+        #[allow(clippy::needless_borrow)]
+        #[allow(clippy::suspicious_else_formatting)]
+        /// Special case for `living::Absorption`.
+        /// Updates the `AbsorptionAmount` component of the player entity.
+        fn update_living_and_player_absorption(
+            mut query: Query<(&living::Absorption, &mut player::AbsorptionAmount), Changed<living::Absorption>>
```

Comment:
What's the difference between `living::Absorption` and `player::AbsorptionAmount`? Feels like the same thing.

Reply:
`player::AbsorptionAmount` is a **tracked** field (so updated via the `EntityTrackerUpdateS2c` packet), but `living::Absorption` is NOT tracked. `player::AbsorptionAmount` is only tracked to update the client on their own absorption level. Furthermore, `player::AbsorptionAmount` is **only** for the client, not for any other entity.